src/index.js

```diff
@@ -6,14 +6,21 @@ import App from './App';
 import reportWebVitals from './reportWebVitals';
 import './style/fonts.css';
 import Top from './components/Top/Top';
-import { BrowserRouter } from 'react-router-dom';
-
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import FetchHotelsRouter from '../src/components/filter/HotelId/NewHotels';
 
 
 ReactDOM.render(
   <BrowserRouter>
     <Top />
+    <Switch>
+        <Route exact patch = "/">
     <App />
+        </Route>
+        <Route exact patch = "hotels/{hotel-id}">
```

Comment:
path, not patch.
Also take a look at https://v5.reactrouter.com/web/api/Hooks/useparams and see how the parameters are specified